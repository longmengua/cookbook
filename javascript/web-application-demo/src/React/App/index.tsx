import React from "react";
import "./index";
import "./index_c";

function App() {
	return (
		<div className="App">
			Index start
		</div>
	);
}

export default App;
