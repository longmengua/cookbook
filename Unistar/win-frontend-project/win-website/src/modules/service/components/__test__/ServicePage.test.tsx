import {shallow} from "enzyme";
import React from "react";
import ServicePage from "../ServicePage";

describe("<ServicePage>", () => {
    it("take a snapshot", () => {
        const component = shallow(<ServicePage />);
        expect(component).toMatchSnapshot();
    });
});
