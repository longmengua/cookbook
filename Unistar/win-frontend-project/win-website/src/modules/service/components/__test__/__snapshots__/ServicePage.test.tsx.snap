// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ServicePage> take a snapshot 1`] = `
<CommonTemplates>
  <div
    className="sub-bg"
  >
    <div
      className="container"
    >
      <div
        className="lottery-list-wrapper"
      >
        <div
          className="lottery-award-content"
        >
          <h3>
            服务项目说明
          </h3>
          <div
            className="service-info-menu"
          >
            <ul>
              <li
                className="active"
              >
                <div
                  className="service01"
                />
                <h3>
                  即时开奖
                </h3>
              </li>
              <li>
                <div
                  className="service02"
                />
                <h3>
                  开奖记录
                </h3>
              </li>
              <li>
                <div
                  className="service03"
                />
                <h3>
                  权威奖源
                </h3>
              </li>
              <li>
                <div
                  className="service04"
                />
                <h3>
                  奖源接口
                </h3>
              </li>
              <li>
                <div
                  className="service05"
                />
                <h3>
                  接口调用
                </h3>
              </li>
            </ul>
          </div>
          <div
            className="service-info-content"
          >
            <div
              className="text-content"
            >
              <p>
                奖源8网为彩票行业客户提供即时、安全、稳定的彩票开奖服务，以及全面的历史数据查询与推送服务。官方奖源，公信可靠。涵盖重庆时时彩（欢乐生肖）、上海11选5、淘宝两分彩等在内国内外各大热门官方彩种，高响应无延迟，安全防劫持，权威可靠！
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</CommonTemplates>
`;
